config/demo.exs

```diff
@@ -0,0 +1,31 @@
+import Config
+
+config :trento, TrentoWeb.Endpoint,
+  check_origin: :conn,
+  cache_static_manifest: "priv/static/cache_manifest.json",
+  server: true
+
+config :swoosh, local: false
+
+# Do not print debug messages in production
+# config :logger, level: :info
+
+config :trento, Trento.Scheduler,
+  jobs: [
+    clusters_checks_execution: [
+      schedule: {:extended, "@hourly"}
+    ],
+    heartbeat_fake: [
+      schedule: {:extended, "*/5"},
+      task: {Trento.Heartbeats.Faker, :send_heartbeats, []},
```

Comment:
Just as a naming convention, it looks strange that we use `Faker` here and the `Mock` suffix in the others hehe

Reply:
This was intentional, there are some differences in the terms and @fabriziosestito and I agreed that "faker" was probably a better term here

Reply:
oh i missed this, i think `Fake` is the right naming for this kind of stuff.
The Mock runner we use in dev should be a `Fake` as well. 

See: https://stackoverflow.com/questions/346372/whats-the-difference-between-faking-mocking-and-stubbing